docs/feature_flag_configuration.md

```diff
@@ -6,24 +6,51 @@ The `FeatureFlagConfiguration` version `v1alpha2` CRD defines the a CR with the
 apiVersion: core.openfeature.dev/v1alpha2
 kind: FeatureFlagConfiguration
 metadata:
-    name: featureflagconfiguration-sample
+  name: featureflagconfiguration-sample
 spec:
-    flagDSpec:
-        envs:
-        - name: FLAGD_PORT
-          value: "8080"
-    featureFlagSpec: 
-        flags: 
-            foo: 
-                state: "ENABLED"
-                variants: 
-                    bar: "BAR"
-                    baz: "BAZ"
-                defaultVariant: "bar"
+  flagDSpec:
```

Comment:
I think `flagDSpec` is deprecated now. We should remove this example from the docs.

Reply:
ok